packages/contracts/contracts/newsroom/Newsroom.sol

```diff
@@ -2,18 +2,14 @@ pragma solidity ^0.4.19;
 import "./ACL.sol";
 
 contract Newsroom is ACL {
-  event ContentProposed(address indexed author, uint indexed id);
-  event ContentApproved(uint id);
-  event ContentDenied(uint id);
+  event ContentAdded(address indexed editor, uint indexed id);
   event NameChanged(string newName);
 
   string private constant ROLE_REPORTER = "reporter";
   string private constant ROLE_EDITOR = "editor";
 
   uint private latestId;
-  mapping(uint => Content) private content;
-  mapping(uint => bool) private waiting;
-  mapping(uint => bool) private approved;
+  mapping(uint => Revision) private content;
```

Comment:
we can make this mapping public, actually, and get rid of the getters (see PR 86 changes to Listing/Challenge/Appeal mappings)